modules/rosa-sdpolicy-account-management.adoc

```diff
@@ -45,7 +45,7 @@ All {product-title} clusters support a maximum of 180 worker nodes.
 The `Default` machine pool node type and size cannot be changed after the cluster is created.
 ====
 
-Control plane and infrastructure nodes are deployed and managed by Red Hat. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss. There are at least 3 control plane nodes that handle etcd- and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. Control plane and infrastructure nodes are strictly for Red Hat workloads to operate the service, and customer workloads are not permitted to be deployed on these nodes.
+Control plane and infrastructure nodes are deployed and managed by Red Hat. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss. There are at least 3 control plane nodes that handle etcd- and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes, including Red Hat workloads, such as Red Hat Quay, Red Hat Advanced Cluster Management, Red Hat Advanced Cluster Security, Red Hat OpenShift Service Mesh, Red Hat OpenShift Serverless, Red Hat OpenShift Logging, Red Hat OpenShift Pipelines, etc.
```

Comment:
Should we use the attributes here? 

For example, :pipelines-title: for Red Hat OpenShift Pipelines?